src/example-types/facet.js

```diff
@@ -134,5 +138,75 @@ module.exports = {
           }),
         options
       ),
-    })),
+    }))
+
+    let missedValues = _.difference(
+      values,
+      _.map(
+        ({ name }) => (node.isMongoId ? _.toString(name) : name),
+        results.options
+      )
+    )
+
+    let getMissedValues = (node, missedValues) =>
+      node.isMongoId ? _.map(ObjectID, missedValues) : missedValues
```

Comment:
`getMissedValues` isn't what this does - it's really a `maybeMapObjectId`. Since `node` is already in scope here, you can simplify to something like this:

```js
let maybeMapObjectId = F.when(node.isMongoId, _.map(ObjectId))
```